Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN mkdir -p  /var/log/marketplace
 WORKDIR /opt/snet/snet-marketplace-service
 COPY . .
 RUN pip3 install -r parse_events/requirement.txt
-RUN npm install
```

Comment:
@vinthedark we will push this change to master tommorow to mainnet? please verify it will not break anything